### PR TITLE
Fix Missed PsyCpAirFnWTb Calls

### PR DESCRIPTION
**Pull request by mitchute**
PR fixes a few missed calls and documentation that were missed in #7479. 

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

**Comment by mitchute** [comment]
@Nigusse @Myoldmopar The ```PsyCpAirFnWTdb``` and ```_fast``` functions still exist in [Pychrometrics.hh](https://github.com/NREL/EnergyPlus/blob/b1c11ff5a75e8e34a6e667b25b8d350f4f1cddae/src/EnergyPlus/Psychrometrics.hh#L379). The ```_fast``` function can be deleted, IMO, since it is not used anywhere. However, the other is only used for comparison against ```PsyCpAirFnW``` in the unit tests. What do you think about moving that function to the unit test file so that there's no confusion about which function to use?

**Comment by mitchute** [comment]
As expected from #7479, there are a few small math diffs due to the method change. Once we resolve what to do with the old ```PsyCpAirFnWTdb``` function, this should be ready.

**Comment by mitchute** [comment]
Ready for review.

File: doc/module-developer/src/energyplus-services/psychrometric-services.tex [hunk 1]
```diff
@@ -44,7 +44,7 @@ \subsection{PsyRhoAirFnPbTdbW (Pb,Tdb,W,calledfrom)}\label{psyrhoairfnpbtdbw-pbt
 
 Returns the density of air in kilograms per cubic meter as a function of barometric pressure {[}Pb{]} (in Pascals), dry bulb temperature {[}Tdb{]} (in Celsius), and humidity ratio {[}W{]} (kilograms of water per kilogram of dry air).
 
-\subsection{PsyCpAirFnWTdb (W,Tdb,calledfrom)}\label{psycpairfnwtdb-wtdbcalledfrom}
+\subsection{PsyCpAirFnW (W,calledfrom)}\label{psycpairfnw-wcalledfrom}
 
 Returns the specific heat of air in Joules per kilogram degree Celsius as a function of humidity ratio {[}W{]} (kilograms of water per kilogram of dry air) and dry bulb temperature {[}Tdb{]} (Celsius).
```
Comment (Nigusse) [inline doc/module-developer/src/energyplus-services/psychrometric-services.tex:49]: This needs removing  the text "dry bulb temperature {[}Tdb{]} (Celsius)"

File: tst/EnergyPlus/unit/HVACUnitarySystem.unit.cc [hunk 2]
```diff
@@ -3360,7 +3360,7 @@ TEST_F(EnergyPlusFixture, UnitarySystem_GetInput)
     SimUnitarySystem(UnitarySystem(1).Name, FirstHVACIteration, UnitarySystem(1).ControlZoneNum, ZoneEquipList(1).EquipIndex(1), _, _, _, _, true);
 
     ZoneTemp = Node(ControlZoneNum).Temp;
-    CpAir = PsyCpAirFnWTdb(Node(InletNode).HumRat, Node(InletNode).Temp);
+    CpAir = PsyCpAirFnW(Node(InletNode).HumRat, Node(InletNode).Temp);
```
Comment (Nigusse) [inline tst/EnergyPlus/unit/HVACUnitarySystem.unit.cc]: Need to remove "Node(InletNode).Temp" argument.

File: tst/EnergyPlus/unit/HVACUnitarySystem.unit.cc [hunk 3]
```diff
@@ -3396,7 +3396,7 @@ TEST_F(EnergyPlusFixture, UnitarySystem_GetInput)
...
```
Comment (Nigusse) [inline tst/EnergyPlus/unit/HVACUnitarySystem.unit.cc]: Same as above.

File: tst/EnergyPlus/unit/HVACUnitarySystem.unit.cc [hunk 4]
```diff
@@ -4176,7 +4176,7 @@ TEST_F(EnergyPlusFixture, UnitarySystem_VarSpeedCoils)
...
```
Comment (Nigusse) [inline tst/EnergyPlus/unit/HVACUnitarySystem.unit.cc]: Same as above.

File: tst/EnergyPlus/unit/HVACUnitarySystem.unit.cc [hunk 5]
```diff
@@ -4645,7 +4645,7 @@ TEST_F(EnergyPlusFixture, UnitarySystem_VarSpeedCoils_CyclingFan)
...
```
Comment (Nigusse) [inline tst/EnergyPlus/unit/HVACUnitarySystem.unit.cc]: Same as above.

File: tst/EnergyPlus/unit/HVACUnitarySystem.unit.cc [hunk 6]
```diff
@@ -8754,7 +8754,7 @@ TEST_F(EnergyPlusFixture, UnitarySystem_ASHRAEModel_WaterCoils)
...
```
Comment (Nigusse) [inline tst/EnergyPlus/unit/HVACUnitarySystem.unit.cc]: Same as above. There are a few places that need removing the temp argument from the PsyCpAirFnW() function.

File: tst/EnergyPlus/unit/HVACUnitarySystem.unit.cc [hunk 7]
```diff
@@ -4208,7 +4208,7 @@ TEST_F(EnergyPlusFixture, UnitarySystem_VarSpeedCoils)
...
```
Comment (Nigusse) [inline tst/EnergyPlus/unit/HVACUnitarySystem.unit.cc]: Same as above.

File: tst/EnergyPlus/unit/HVACUnitarySystem.unit.cc [hunk 8]
```diff
@@ -4606,7 +4606,7 @@ TEST_F(EnergyPlusFixture, UnitarySystem_VarSpeedCoils_CyclingFan)
...
```
Comment (Nigusse) [inline tst/EnergyPlus/unit/HVACUnitarySystem.unit.cc]: Same as above.

**Comment by rraustad** [comment]
The unit test HVACUnitarySystem.unit.cc is no longer used and should be deleted from the repository. I forgot to do this during refactoring.

**Comment by mitchute** [comment]
Thanks @rraustad. I'll get it taken care of.

**Comment by mitchute** [comment]
@Nigusse Thanks for reviewing this. Feel free to merge it if CI looks good when it finishes.

**Comment by Nigusse** [comment]
@mitchute I looked at the results and I see a few small diffs that look OK, but there are four test files that failed which should not. These four files have ems program. I think these failed test files need to be resolved before merging.

**Comment by mitchute** [comment]
@Nigusse I see two files with EDD diffs, but those are purely text diffs due to changing the EMS method's signature. However, I don't see anything else. Which files are you referring to?

**Comment by Nigusse** [comment]
@mitchute I checked the latest runs, all is good excepts the warnings, which are text diffs as you pointed out rightly. My earlier comment must have been based on previous runs. This is good to merge!

**Comment by Nigusse** [comment]
@mitchute I will merge this branch shortly.

**Comment by mjwitte** [comment]
Hmm, just catching up here. Did we add a transition rule for the old function name to new name or is EMS just mapping the old one behind the scenes?

**Comment by Myoldmopar** [comment]
I don't believe a transition rule was written, so I think the Erl processor must just be mapping to the new function behind the scenes.  I'm not saying we shouldn't fix it, but I believe that's what it is doing.

**Comment by mitchute** [comment]
Missed that too... I'll get that ready.

**Pull request merged**